src/pybind/mgr/dashboard/controllers/health.py

```diff
@@ -92,6 +92,13 @@
             'up': (int, ''),
         }], '')
     }, ''),
+    'perf_stat': ({
+        'osd_perf': ([{
+            'osd': (int, ''),
+            'commit_latency_ms': (int, ''),
```

Comment:
Thanks ! @VallariAg This is a Nice improvement for OSD performance,
Can you please explain more about what is exactly _commit_latency_ and _apply_latency_ in osd's?
and I prefer Bar chart it looks good and easy to understand.

Reply:
Hey @sunilangadi2, 
When we carry out an operation like `rados bench -p testbench 10 write --no-cleanup`, the command `ceph osd perf` gives the following stats:
```
osd  commit_latency(ms)  apply_latency(ms)
  2                 154                154
  0                 143                143
  1                 168                168
```
Here, the commit latency is the time taken to commit the ongoing operation and the apply latency is the time taken to apply the changes to disks. 

Indeed, the bar chart looks like a good solution! I'm done with it's implementation, I'm currently looking into the grafana charts for maybe a more engaging solution.